### ISSUE TITLE
NoSuchFileException in KubeconfigWatchService

### ISSUE THREAD

**Issue by etanshaul** [GoogleCloudPlatform/cloud-code-intellij#2774]
20.5.1
IDEA 2020.1.1

```
java.nio.file.NoSuchFileException: /**/**/.kube
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:92)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at java.base/sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at java.base/sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:149)
	at java.base/java.nio.file.Files.readAttributes(Files.java:1763)
	at java.base/sun.nio.fs.PollingWatchService.doPrivilegedRegister(PollingWatchService.java:166)
	at java.base/sun.nio.fs.PollingWatchService$2.run(PollingWatchService.java:147)
	at java.base/sun.nio.fs.PollingWatchService$2.run(PollingWatchService.java:144)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/sun.nio.fs.PollingWatchService.register(PollingWatchService.java:143)
	at java.base/sun.nio.fs.UnixPath.register(UnixPath.java:920)
	at java.base/java.nio.file.Path.register(Path.java:894)
	at com.google.cloud.tools.intellij.kubernetes.configs.KubeconfigWatchService$watchForUpdates$1.invokeSuspend(KubeconfigWatchService.kt:55)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(Dispatched.kt:241)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:594)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.access$runSafely(CoroutineScheduler.kt:60)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:740)
```

**Comment by etanshaul** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-intellij-internal](https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/issues/1935)

**Comment by etanshaul** [comment]
fixed via `20.6.1`
